src/api/controllers/articleController.js

```diff
@@ -0,0 +1,71 @@
+import _ from 'lodash';
```

Comment:
@nkpremices shouldn't this file be called `articleRatingsController?`

Reply:
Oh yes!! definitely. 
Let me rename it @MemunaHaruna